### ISSUE TITLE
GDCMImageIO in master branch crash at ultrasound files after PR 1306

### ISSUE THREAD

**Issue by issakomi** [InsightSoftwareConsortium/ITK#1308]
Try any lossy JPEG YBR_FULL_422 file, e.g. from [www.creatis.insa-lyon.fr](https://www.creatis.insa-lyon.fr/~jpr/PUBLIC/gdcm/gdcmSampleData/US_DataSet/Acusson_US/) CLIP0001
```
WARNING: In /home/r/itk/myITK/ITK/Modules/IO/GDCM/src/itkGDCMImageIO.cxx, line 314
GDCMImageIO (0x55fe83dc09b0): Converting from integer YBR to integer RGB is a lossy operation.

Not a JPEG file: starts with 0x00 0x00
itk::ERROR: GDCMImageIO(0x55fe83dc09b0): Failed to get the buffer!
```

**Comment by issakomi** [comment]
There are two solutions currently available. Closed.

**Comment by dzenanz** [comment]
The issue should be closed after a fix has been merged.

**Comment by issakomi** [comment]
That bug was mentioned [here](https://github.com/InsightSoftwareConsortium/ITK/pull/1240#issuecomment-531471129) and [here](https://github.com/InsightSoftwareConsortium/ITK/pull/822#issuecomment-489996320).

**Comment by seanm** [comment]
CC @seanm @vfonov

**Comment by dzenanz** [comment]
Fixed by #1309.
